operations/power_management/Prepare_the_System_for_Power_Off.md

```diff
@@ -6,331 +6,356 @@ The `sat bootsys shutdown` and `sat bootsys boot` commands are used to shut down
 
-       ```bash
-       ncn# cray cfs components describe XNAME | grep bos_session
-       ```
+      ```bash
+      ncn# cray cfs components describe XNAME | grep bos_session
+      ```
 
-       Example output:
+      Example output:
 
-       ```
-       bos_session = "e98cdc5d-3f2d-4fc8-a6e4-1d301d37f52f"
-       ```
+      ```text
+      bos_session = "e98cdc5d-3f2d-4fc8-a6e4-1d301d37f52f"
+      ```
 
-    1. Find the required `templateUuid` value with BOS.
+   1. Find the required `templateUuid` value with BOS.
 
-       ```bash
-       ncn# cray bos session describe BOS_SESSION | grep templateUuid
-       ```
+      ```bash
+      ncn# cray bos session describe BOS_SESSION | grep templateName
+      ```
 
-       Example output:
+      Example output:
 
-       ```
-       templateUuid = "compute-nid1-4-sessiontemplate"
-       ```
+      ```text
+      templateName = "cos-2.3.101"
+      ```
 
-    1. Determine the list of xnames associated with the desired boot session template.
+   1. Determine the list of xnames or role groups or HSM groups associated with the desired boot session template.
 
-       ```bash
-       ncn# cray bos sessiontemplate describe SESSION_TEMPLATE_NAME | grep node_list
-       ```
+      ```bash
+      ncn# cray bos sessiontemplate describe SESSION_TEMPLATE_NAME | egrep "node_list|node_roles_groups|node_groups"
+      ```
 
-       Example output:
+      Example output:
 
-       ```
-       node_list = [ "x3000c0s19b1n0", "x3000c0s19b2n0", "x3000c0s19b3n0", "x3000c0s19b4n0",]
-       ```
+      ```text
+      node_roles_groups = [ "Compute",]
+      ```
 
-1.  Use sat to capture state of the system before the shutdown.
+1. Use sat to capture state of the system before the shutdown.
 
-    ```bash
-    ncn# sat bootsys shutdown --stage capture-state
-    ```
+   ```bash
+   ncn# sat bootsys shutdown --stage capture-state
+   ```
 
-1.  Optional system health checks.
+1. Optional system health checks.
 
-    1.  Use the System Diagnostic Utility (SDU) to capture current state of system before the shutdown.
+   **Important:** Running the System Diagnostic Utility (SDU) may take 20 to 60 minutes to run so
+   start this command in one terminal session and then start another session for the other optional
+   system health checks to be run while SDU is in progress.
 
-        **Important:** SDU takes about 15 minutes to run on a small system \(longer for large systems\).
+   1. Use the System Diagnostic Utility (SDU) to capture current state of system before the shutdown.
 
-        ```bash
-        ncn# sdu --scenario triage --start_time '-4 hours' \
-                 --reason "saving state before powerdown"
-        ```
+      ```bash
+      ncnm# sdu --scenario triage --start_time '-4 hours' \
+               --reason "saving state before powerdown"
+      ```
 
-    1.  Capture the state of all nodes.
+      ***Important:** If the master node where you attempt to run `sdu` does not have SDU installed or
```

Comment:
```suggestion
      ***Important:*** If `sdu` is not installed on a particular master node, then it may be installed on a different master node.
```